cmd/snapd-apparmor/main_test.go

```diff
@@ -52,6 +53,15 @@ func (s *mainSuite) TearDownTest(c *C) {
 	dirs.SetRootDir("/")
 }
 
+func MockWSL(onWSL bool) (restorer func()) {
+	oldWSL := release.OnWSL
+
+	release.OnWSL = onWSL
+	return func() {
+		release.OnWSL = oldWSL
+	}
```

Comment:
nitpick: you can use `testutil.Backup()` here

Reply:
Good point. Fixed.